Services/source/logger/logger_service.c

```diff
@@ -51,13 +52,40 @@ int file_exists(const char *filename){
     return 1;
 }
 
+SAT_returnState get_file(const char *filename, csp_packet_t *packet) {
+    int8_t status;
+    int32_t file;
+    uint32_t data_size;
+    char* log_data[500] = {0};
+    if (file_exists(filename) == 0) {
+        file = red_open(filename, RED_O_RDONLY);
+        if(file > -1){
+            data_size = red_read(file, log_data, max_file_size);
+            if (data_size == 0){
+                status = -1;
+                strncpy(log_data, "Log file is empty\n", max_string_length);
+            } else {
+                status = 0;
+            }
+        } else {
+            status = -1;
+            strncpy(log_data, "Can't open log file\n", max_string_length);
```

Comment:
Can we get the reliance edge error no. here into the log

Reply:
ok I added that and tested it. works well